### ISSUE TITLE
Suggestion: do not truncate article titles

### ISSUE THREAD

**Issue by toriningen** [border-radius/hn-best-comments#2]
Currently titles are truncated way too early (at 50 characters), making it quite hard to follow the context without following the link first.

Out of 530 non-duped article titles, 304 (>57%) are longer than 50 characters, 86 being the longest so far.

Would it make sense to assume 100 characters as the new limit?

**Comment by border-radius** [comment]
There is no character limit in the bot itself, bot just parsing /bestcomments page from Hacker News and titles here are already truncated. I think fetching extra page for every comment just to parse title will be overhead so I decided to keep original HN format

**Comment by border-radius** [comment]
Fixed in new version
